protocol/messenger.go

```diff
@@ -3604,6 +3607,9 @@ func (m *Messenger) handleRetrievedMessages(chatWithMessages map[transport.Filte
 			}
 
 			statusMessages, acks, err := m.sender.HandleMessages(shhMessage)
+			if m.telemetryClient != nil && err != nil {
+				go m.telemetryClient.UpdateEnvelopeProcessingError(shhMessage, err)
```

Comment:
Here, in case of error, I send envelope data and processing error to the service.